dbt_subprojects/hourly_spellbook/models/_sector/perpetual/projects/avantis/avantis_base_perpetual_trades.sql

```diff
@@ -0,0 +1,46 @@
+{{ config(
+    schema = 'avantis_perpetual_trades',
+    alias = 'perpetual_trades',
+    post_hook='{{ expose_spells(blockchains = \'["base"]\',
+                                    spell_type = "project",
+                                    spell_name = "avantis",
+                                    contributors = \'["princi"]\') }}'
+        )
+}}
+
+{% set avantis_base_perpetual_trade_models = [
+    ref('avantis_v1_base_perpetual_trades')
+] %}
+
+SELECT *
+FROM
+(
+    {% for avantis_perpetual_trades in avantis_base_perpetual_trade_models %}
+    SELECT
+        blockchain
+		,block_date
```

Comment:
Please replace tabs with 4 spaces.
```suggestion
        blockchain
        ,block_date
```